### DISCUSSION TITLE
mit18.330: Introduction to numerical analysis - CS%E8%87%AA%E5%AD%A6%E6%8C%87%E5%8D%97

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#171] [Announcements]
# mit18.330: Introduction to numerical analysis - CS%E8%87%AA%E5%AD%A6%E6%8C%87%E5%8D%97

CS自学指南

https://csdiy.wiki/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/numerical/

**Comment by 2060490154** [comment]
github只看到有课程的pdf文件，视频在哪没看到

**Comment by Practicing7** [comment]
这个课程没有视频吗，难道要看ppt学习 难度好大

**Comment by novembbbbber** [comment]
https://www.dropbox.com/sh/daxupg7ab1546y4/AABpYpYGUk-jqzAweiUlzMrta?dl=0
dropbox里面有大部分课程，不过要梯子，而且注册dropbox比较麻烦，要花些时间。

**Reply by novembbbbber** [reply]
直接在链接里面看好像只能看一小部分，最好用dropbox账号

**Reply by IndifferentArea** [reply]
您好，请问方便说下这个dropbox链接从哪里得到的吗，是否可以通过类似的方式看到别的课程呢👀

**Reply by novembbbbber** [reply]
这个是我直接在B站上搜mit18.330，然后只有2个视频，不过视频简介里面有dropbox的链接。我也想把dropbox里面的视频发布到b站上，不过我觉得要是先把视频下载下来再上传的话太麻烦了

**Reply by novembbbbber** [reply]
关于dropbox里面的视频，那个lecture15我觉得好像有点不对劲，时长有点短，课程具体内容我不是很了解，不知道有没有问题。

**Reply by IndifferentArea** [reply]
thx

**Reply by novembbbbber** [reply]
【mit18.330lecture4】 https://www.bilibili.com/video/BV1pP411u7KE/?share_source=copy_web&vd_source=26e90d756a6cbf8a5f9b827f9ba8ee8f

**Reply by jahox0102** [reply]
感谢分享，dropbox现在里的项目已经失效了，b站搬运还可以观看

**Comment by Practicing7** [comment]


**Reply by novembbbbber** [reply]
关于dropbox里面的视频，那个lecture15我觉得好像有点不对劲，时长有点短，课程具体内容我不是很了解，不知道有没有问题。

**Comment by shuoyue21** [comment]
MIT18.330 Introduction to numerical analysis.7z
链接：https://pan.quark.cn/s/cd36658121a2
spring 20的录播,解压密码jdj*XCJ.dez3ehp@atu
转的github上的drop网盘

**Reply by JinYu1998** [reply]
链接里的文件没有了

**Reply by Radbruch** [reply]
链接失效了，可以麻烦您再发一次吗

**Reply by haozhihan** [reply]
这个密码好像不太对啊

**Reply by Yyugust** [reply]
链接失效了，可以麻烦您再发一次吗？非常感谢！

**Reply by shuoyue21** [reply]
已更新

**Reply by calinear** [reply]
密码不对、、

**Comment by TephrocactusMYC** [comment]
巨神是怎么做到连这些数学课都学了这么多的，太恐怖了吧

**Reply by MTDickens** [reply]
说实话我们学校（浙江大学）除了信息论以外，其它所有这上面的数学课都要修。其实也不算很多了。有些学校还必修抽象代数，比这列表上还要多。

**Comment by MTDickens** [comment]
这里有最新的视频：https://www.dropbox.com/sh/ubkqwrqxnukgllc/AAA2cH9r7YQL7WmYVt-bblxta?dl=0

**Comment by dghtucs** [comment]
建了一个讨论群，可入QQ：818093454

**Comment by NengxinMou** [comment]
没有Julia这门语言的基础能直接学吗？还是说要有一定的基础呢？QAQ

**Reply by MTDickens** [reply]
Julia 现学现用就好，也不是什么难学的语言（）

**Comment by lil-Journey** [comment]
自建日本节点服务器 月租5元不限流量 本人也在用 失效时可联系人工修复 qq:1165116287
